Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -151,7 +150,12 @@ def audit
 
     new_formula_problem_lines = []
     ff.sort.each do |f|
-      options = { new_formula: new_formula, strict: strict, online: online, only: args.only, except: args.except }
+      if only_cops
```

Comment:
```ruby
only = only_cops ? ["style"] : args.only
```

Reply:
I haven't seen ternary operator being used in the codebase so I thought its against our style :-|

Reply:
I see it in use a fair bit:

* https://github.com/Homebrew/brew/blob/master/Library/Homebrew/dev-cmd/audit.rb#L240
* https://github.com/Homebrew/brew/blob/master/Library/Homebrew/dev-cmd/audit.rb#L245
* https://github.com/Homebrew/brew/blob/master/Library/Homebrew/dev-cmd/audit.rb#L970

Reply:
Cool. Updating the code.

Reply:
I think it's personal preference at this point if it's simple; personally I find the ternary operator always harder to read but this is fine either way.